### ISSUE TITLE
Manage labels to fit our current need

### ISSUE THREAD

**Issue by kaichih** [OpenIoMeT/iomet#7]
We need to manage "labels" to label our issues in our Github issue here. I will propose some labels here, and need your reviews later.

**Comment by gangit** [comment]
Hi All,

Thank for the sharing.

Just to let know that an interesting presentation is actually live stream
on some e-health projects at

*http://www.odess.io/streaming-en.html
http://www.odess.io/streaming-en.html *

_Best,_

_ganesh_

**Comment by kumarrishav** [comment]
Thanks for filling this @kaichih .

Sorry @gangit  for late update, we are combining all the stuff and resources what we planned during London all hands. We will update you with all the new details very soon. Thanks for the patience.

**Comment by gangit** [comment]
Thanks Kumar...

No worries

Best

**Comment by snowmantw** [comment]
I suggest that we at least need some functional labels like:
1. **discussions**: for discussions like this, rather than implementation issues from those decisions already made. So by default, any issue without this label would be implementation issue, which mean they are expected to be solved within reasonable period. For example, 3 days.
2. **meta**: once we have some plans or milestones, no matter whether they're technical or not, we may need a **meta** label to mark it for tracking and further discussions. This kind of issues are not expected to be solved but should come with dependencies. 
3. **wontfix**, **duplicate**, **assigned**, **invalid**

And we could take a look at this reference as well:

https://robinpowered.com/blog/best-practice-system-for-organizing-and-tagging-github-issues/

Besides that, after we have more clear model about how this project (and its sub-projects) works, we should have regular triage to update progress and scan all unnoticed issues.

**Comment by snowmantw** [comment]
Also, we may need some bots to work on labels automatically. For example, the triaging could be helped by automatically collecting unassigned implementation issues.

**Comment by snowmantw** [comment]
So @kaichih , since you own this issue, you can just create all the labels and close this issues. For the bots I would fire another issue.

**Comment by kaichih** [comment]
I was thinking to use labels to represent the timeline and use it to manage our issues. But, later, I think it's better to use milestone. I will create another issue to explain it to you guys.

**Comment by kaichih** [comment]
Due to #12, this should be resolved in milestone 1.

**Comment by kumarrishav** [comment]
Purpose of labels can be different. Labels Like:  Discussion wanted, bug, feature, documentation, easy bug,Q/A wanted, UI/UX/Design help etc. So that we can use this label for the issues (if required i.e each issue doesn't need to have a label)

**Comment by kaichih** [comment]
Yes, label should be like what you said, and it is used to indicate what kinds of issues they are.
